File: src/main/java/interview/greedy/MinimumAbsoluteDifferenceInAnArray/minimum_absolute_difference_in_an_array.jl


# Complete the minimumAbsoluteDifference function below.
function minimumAbsoluteDifference(arr)


end

fptr = open(ENV["OUTPUT_PATH"], "w")

n = parse(Int32, readline(STDIN))

arr = [parse(Int32, k) for k in split(readline(STDIN))]

result = minimumAbsoluteDifference(arr)

write(fptr, string(result)*"\n")

close(fptr)

File: src/main/java/interview/greedy/MinimumAbsoluteDifferenceInAnArray/minimum_absolute_difference_in_an_array.rb
#!/bin/ruby

require 'json'
require 'stringio'

# Complete the minimumAbsoluteDifference function below.
def minimumAbsoluteDifference(arr)


end

fptr = File.open(ENV['OUTPUT_PATH'], 'w')

n = gets.to_i

arr = gets.rstrip.split(' ').map(&:to_i)

result = minimumAbsoluteDifference arr

fptr.write result
fptr.write "\n"

fptr.close()
